Example/Podfile

```diff
@@ -1,4 +1,3 @@
-source 'https://github.com/CocoaPods/Specs.git'
```

Comment:
I think this might cause issues with loading correct version tho? unless all dependencies will points to the branch/commit?

Reply:
1. this is in Example
2. seems not a problem, let me test it in app